docs/src/main/paradox/solr.md

```diff
@@ -0,0 +1,169 @@
+# Solr Connector
+
+The Solr connector provides Akka Stream sources and sinks for Solr.
+
+For more information about Solr please visit the [Solr documentation](http://lucene.apache.org/solr/resources.html).
+
+
+### Reported issues
+
+[Tagged issues at Github](https://github.com/akka/alpakka/labels/p%3Asolr)
+
+
+## Artifacts
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-solr_$scalaBinaryVersion$
+  version=$version$
+}
+
+## Set up client
+
+Sources, Flows and Sinks provided by this connector need a prepared `org.apache.solr.client.solrj.SolrClient` to
+access to Solr.
```

Comment:
Thank you for your reviewing, I don't think this is a strict requirement. Especially, `client.close()` based on exceptions and it can be used out of the actor system in the application.